app/assets/stylesheets/course/achievement.scss

```diff
@@ -1,3 +1,13 @@
 // Place all the styles related to the course/achievement controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.course-achievements {
+  .achievement-list,
+  .achievement-form {
+    .image > img {
+      height: 75px;
```

Comment:
I think all the other places where we use achievement images should change to use a sass variable for determining size.